docs/data/joy/components/modal/modal.md

```diff
@@ -8,113 +8,139 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/
 
 # Modal
 
-<p class="description">The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, or whatever else.</p>
-
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+<p class="description">The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, and other interactive elements.</p>
```

Comment:
Capital M since we're referring specifically to this lib's component (so, proper noun)

```suggestion
<p class="description">The Modal component provides a solid foundation for creating dialogs, popovers, lightboxes, and other interactive elements.</p>
```